_posts/fork/2021-03-22-OrangeheartMonokai.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: theme
+title: OrangeheartMonokai
+category: theme
```

Comment:
We need to change this from theme to fork as well.

And maybe put a link in https://theme.typora.io/theme/OrangeHeart/